### PR TITLE
Update main_config.yaml.sample file and helper commands

### PR DESCRIPTION
**Pull request by npamudika**
This PR updates main_config.yaml.sample file with the correct endpoints and fixes issues in helper commands.

### CODE REVIEW

File: import-export-cli/box/resources/sample/sample_config.yaml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-    admin_endpoint: "https://localhost:9443/api/am/admin/v0.16"
-    api_list_endpoint: "https://localhost/publisher/apis"
-    api_manager_endpoint: "https://localhost/apim"
-    application_list_endpoint: "https://localhost:9443/api/am/admin/v0.16/applications"
-    registration_endpoint: "https://localhost/register"
-    token_endpoint: "https://localhost/token"
+    admin_endpoint: https://localhost:9443/api/am/admin/v0.16
+    api_list_endpoint: https://localhost:9443/api/am/publisher/v0.16/apis
+    api_manager_endpoint: https://localhost:9443
+    application_list_endpoint: https://localhost:9443/api/am/store/v0.16/applications
```
Comment (malinthaprasan) [inline import-export-cli/box/resources/sample/sample_config.yaml]: This should come from admin

File: import-export-cli/box/resources/sample/sample_config.yaml [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
 environments:
   sample-env1:
-    admin_endpoint: "https://localhost:9443/api/am/admin/v0.16"
-    api_list_endpoint: "https://localhost/publisher/apis"
-    api_manager_endpoint: "https://localhost/apim"
-    application_list_endpoint: "https://localhost:9443/api/am/admin/v0.16/applications"
-    registration_endpoint: "https://localhost/register"
-    token_endpoint: "https://localhost/token"
+    admin_endpoint: https://localhost:9443/api/am/admin/v0.16
+    api_list_endpoint: https://localhost:9443/api/am/publisher/v0.16/apis
```
Comment (malinthaprasan) [inline import-export-cli/box/resources/sample/sample_config.yaml]: api_list_endpoint: https://localhost:9443/api/am/publisher/v1/apis

File: import-export-cli/box/resources/sample/sample_config.yaml (hunk 1 above)
Comment (malinthaprasan) [reply import-export-cli/box/resources/sample/sample_config.yaml]: application_list_endpoint: https://localhost:9443/api/am/admin/v0.16/applications

File: import-export-cli/box/resources/sample/sample_config.yaml (hunk 2 above)
Comment (npamudika) [reply import-export-cli/box/resources/sample/sample_config.yaml]: Fixed in https://github.com/wso2/product-apim-tooling/pull/190

File: import-export-cli/box/resources/sample/sample_config.yaml (hunk 1 above)
Comment (npamudika) [reply import-export-cli/box/resources/sample/sample_config.yaml]: Fixed in https://github.com/wso2/product-apim-tooling/pull/190

**Pull request merged**